CONTRIBUTING.md

```diff
@@ -22,9 +22,9 @@ It runs one test for each `.cc` file in the `tests/` directory.  (We have additi
 
 The output can be a bit hard to read, but if a test fails, the reason why will be listed after the `ERROR:root:Test failed for xxx` line.
 
-You can select individual tests by listing their filename without extension as arguments
+You can select individual tests by listing them as arguments. Generally this is 'c.test_' or 'cxx.test_' (depending on whether the test in a C or C++ test) followed by the filename without any extension
```

Comment:
The more general pattern is `$suite.test_$filenamenoext`, where `suite` is one of the subdirectories of `tests/`, currently:

* `driver`
* `c`
* `cxx`

We probably don't need to go into that kind of detail, but the proposed wording feels a little _too_ non-committal :)

Maybe something like:
```
You can select individual tests by listing them as arguments. Test names are derived from the file path and name, e.g. `tests/cxx/array.cc` will be named `cxx.test_array`. You can use `python run_iwyu_tests.py --list` to list all available test names.

    ... your example here ...
```

Reply:
Sounds much better, I'll post an update shortly